### DISCUSSION TITLE
Version difference between composer and github?

### DISCUSSION THREAD

**Discussion by MGeurts** [tasteui/tasteui#443] [General]
When running `composer update` it updates TallStackUI to 1.24.5 while the Github project just moved to 1.24.2.
Why is there a gap?

**Comment by devajmeireles** [comment]
On GitHub, we have the latest and most important tag as a release. When you run `composer update` your Composer will take the latest tag which may be greater than the latest version, if we have released a new tag without a release, for minor corrections. I intend to stop doing that.
